src/tokenomics.md

```diff
@@ -49,24 +49,30 @@ head:
 
 In this section you will find all the information on SORA
 tokenomics. Starting with the Token Bonding curve, each subsection
-will cover the main [SORA economy](sora-economy.md) tokens. It is
-important to remember that the [XOR](xor.md) token does not follow
+will cover the main [SORA economy](/sora-economy.md) tokens. It is
+important to remember that the [XOR](/xor.md) token does not follow
 conventional tokenomics of limited supply, instead, its supply is
-elastic, tied to the [Token Bonding Curve (TBC)](tbc.md) and managed
-through [SORA governance](sora-governance.md).
+elastic, tied to the [Token Bonding Curve (TBC)](/tbc.md) and managed
```

Comment:
The tokenomics have changed, and now XOR is not elastic anymore. XOR is now deflationary, and KXOR has elastic supply.

We should reflect that in the wiki as well.
@WRRicht3r @AndreyKortosov